modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -3,6 +3,7 @@
 import com.github.dockerjava.api.command.ExecCreateCmdResponse;
 import com.github.dockerjava.api.command.InspectContainerResponse;
 import com.github.dockerjava.core.command.ExecStartResultCallback;
+import lombok.NonNull;
```

Comment:
In most other locations we use `org.jetbrains.annotations.NotNull;` :man_shrugging:

Reply:
inherited from the getter

will remove since there can be a case where it is null